R/dataframe-frame.R

```diff
@@ -98,6 +112,15 @@ wrap.PlRDataFrame <- function(x, ...) {
   makeActiveBinding("shape", function() self$`_df`$shape(), self)
   makeActiveBinding("height", function() self$`_df`$height(), self)
   makeActiveBinding("width", function() self$`_df`$width(), self)
+  makeActiveBinding(
+    "flags",
+    function() {
+      out <- lapply(self$get_columns(), \(x) x$flags)
+      names(out) <- self$columns
+      out
```

Comment:
I don't think `names(out) <- self$columns` is necessary since the list is originally named.